stdlib/public/Concurrency/Task.swift

```diff
@@ -566,22 +570,31 @@ extension Task {
   }
 }
 
-/// Calls the given closure with the with the "current" task in which this
-/// function was invoked.
+/// Calls a closure with an unsafe handle to current task.
+///
+/// If you call this function from the body of an asynchronous function,
+/// the unsafe task handle passed to the closure is always be non-nil
+/// because an asynchronous function always runs in the context of a task.
+/// However if you call this function from the body of a synchronous function,
+/// the unsafe task handle is `nil`
```

Comment:
```suggestion
/// the unsafe task handle may be `nil`
``` 

feels more right somehow...? maybe not?

Reply:
If I call this method from sync code, and my backtrace was all sync back up the call stack, when would this return a non-nil value?

Reply:
it would be nil then -- if the entire callstack is outside of swift concurrency APIs

Reply:
Thanks. If it's always `nil` in that case, I'll avoid writing "may be":